python/grass/jupyter/interact_display.py

```diff
@@ -166,3 +216,10 @@ def show(self):
         fig.add_child(self.map)
 
         return fig
+
+    def save(self, filename="GRASS_map.html"):
```

Comment:
What use case do you have in mind for the default? Is this trying to mimic the d-command behavior in creating `map.png` by default? How do you plan to document it?

Reply:
Yes, I think I was thinking of the d-command behavior here. `map.png` would probably be a better choice. Use-case-wise, I was imagining users might want to embed their map in a website or create a website from. They create their map then save as an html (i.e. `m.save(filename="map_for_website.html")`).

Reply:
Right, so what is the use case for the default value? In other words, in what circumstances will the user know about the default and write save() while not saying where to save?

Reply:
AH, ok, I see your point now. Having a default is not necessary. The line should be `def save(filename):`.